### ISSUE TITLE
Various errors (2.07)

### ISSUE THREAD

**Issue by eserte** [jhthorsen/mojolicious-plugin-openapi#100]
There seem to be a couple of conditions causing test failures:
```
#   Failed test 'errors: /v: Does not match ipv4 format.'
#   at t/v2-formats.t line 15.
#     Structures begin differing at:
#          $got->[0] = Does not exist
#     $expected->[0] = HASH(0x2f78c20)
# []
```
and
```
#   Failed test 'valid openapi v3 schema'
#   at t/v3.t line 28.
# [JSON::Validator] The optional YAML::XS module is missing or could not be loaded: YAML::XS version 0.67 required--this is only version 0.62 at (eval 138) line 1.
# BEGIN failed--compilation aborted at (eval 138) line 1.
```

**Comment by eserte** [comment]
Also:
```
#   Failed test '200 OK'
#   at t/default-value.t line 40.
#          got: '404'
#     expected: '200'

#   Failed test 'exact match for JSON Pointer "/days"'
#   at t/default-value.t line 40.
#     Structures begin differing at:
#          $got = undef
#     $expected = HASH(0x2faac08)

#   Failed test 'exact match for JSON Pointer "/name"'
#   at t/default-value.t line 40.
#     Structures begin differing at:
#          $got = undef
#     $expected = HASH(0x2fe68d8)
# Looks like you failed 3 tests of 11.
t/default-value.t ..................... 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/11 subtests 
```
... and some other failures.

**Comment by jhthorsen** [comment]
Thanks for the report! I hope many if the issues have been resolved in 2.08.

https://github.com/jhthorsen/mojolicious-plugin-openapi/compare/2.07...2.08
